src/components/SidePanel/SidePanel.js

```diff
@@ -60,7 +60,7 @@ class SidePanel extends React.PureComponent {
             )}
           </PanelHeader>
           <PanelScrollView>
-            <PanelContent>{children}</PanelContent>
+            <PanelContent>{render ? render(opened) : children}</PanelContent>
```

Comment:
```suggestion
            <PanelContent>{render ? render({ opened }) : children}</PanelContent>
```

Perhaps this would be a nicer API, thoughts @bpierre?

Reply:
Yes definitely, just in case we pass something else in the future!

Some other thoughts:

- Should we pass a `status` directly, which could be either `opening`, `closing`, `opened` or `closed`? This last one will be removed when we’ll move to a `Transition` rather than a `Spring`.
- What would you guys think of detecting if `children` is a function, and calling it in that case? That way we wouldn’t need a `render` prop behaving differently than `children`.
- Should we move to a `Transition` directly?

Reply:
Makes sense! 

For thoughts:
* passing a `status` directly makes sense
* react doesn't allow passing functions as children, so will have to stick with render prop
* lets do it, i'll move to a transition to finish off the PR

Reply:
Hey @kushagrasharma, did you get any time to work on these? We could also give you a hand to get this PR ready, let me know what you prefer!